src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -833,6 +839,33 @@ class HookUtils(object):
                            'Error removing file: %s' % (filename))
         return True
 
+    def _execjob_abort_handler(self, event, cgroup, jobutil):
+        """
+        Handler for execjob_abort events.
+        """
```

Comment:
Suggest you remove _execjob_abort_handler() method since it is an exact copy of _execjob_end_handler() method.

Reply:
Agree.